### PR TITLE
Add Attribute ActiveIssue(15844) to WaitChain() Method in class ProcessWaitingTests under System.Diagnostics.Process

### PR DESCRIPTION
**Pull request by Jiayili1**
Add attribute [ActiveIssue(15844)] to disable test method: System.Diagnostics.Tests.ProcessWaitingTests.WaitChain().

### CODE REVIEW

File: src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs [hunk 1]
```diff
@@ -164,6 +164,7 @@ public void WaitForPeerProcess()
         }
 
         [Fact]
+        [ActiveIssue(15844)]
```
Comment (stephentoub) [inline src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs]: This has only failed on Unix, right? We should only disable it there then, e.g. adding TestPlatforms.AnyUnix

Comment (danmosemsft) [inline src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs]: @Jiayili1, Stephen's correct -- please update your PR to do this. (My bad -- I didn't explain TestPlatforms.) You can find examples elsewhere in the code if you search.

**Comment by danmosemsft** [comment]
@Jiayili1 for PR's like this you'll also want to attach the area label (S.D.Process here) and cc the owner - @Priya91 here.

**Comment by Jiayili1** [comment]
@danmosemsft Got it. I will remember to add the area label and cc the owner later.

**Comment by danmosemsft** [comment]
Also let's include a link to the original issue: #15844

**Comment by danmosemsft** [comment]
When all the CI system is green, I will merge this for you.

**Comment by danmosemsft** [comment]
Only failure is. https://github.com/dotnet/corefx/issues/15383

**Pull request merged**